core/src/main/java/com/datastrato/gravitino/authorization/AccessControlManager.java

```diff
@@ -14,7 +14,11 @@
 
 /**
  * AccessControlManager is used for manage users, roles, admin, grant information, this class is an
- * entrance class for tenant management.
+ * entrance class for tenant management. This lock policy about this is as follows: First, admin
+ * operations are prevented by one lock. Then, other operations are prevented by the other lock. For
```

Comment:
Can we explicitly define these two locks for the ease of understanding? Using `synchronized` is a bit hard to understand.

Reply:
Done.